__tests__/endpoints.test.js

```diff
@@ -88,5 +88,38 @@ describe('/api/articles', () => {
         })
     })
 })
+describe('/api/articles/:article_id/comments', () => {
+    test('200 with all the comment in ascending order related to the article.article_id', () => {
+        return request(app).get('/api/articles/3/comments').expect(200)
+        .then((res) => {
+            const comments = res.body
+            expect(comments.length).toBe(2)
+            expect(comments).toBeSortedBy('created_at', { ascending: true })
```

Comment:
This is a different behaviour and so would be suitable extracted into its own test